pkg/ext/stores/tokens/tokens.go

```diff
@@ -307,14 +317,64 @@ func (t *Store) List(
 	return t.list(ctx, options)
 }
 
-// ConvertToTable implements [rest.Lister], the interface to support the `list` verb.
+// ConvertToTable implements [rest.Lister]/[rest.TableConvertor], the interface to support the `list` verb.
 func (t *Store) ConvertToTable(
 	ctx context.Context,
 	object runtime.Object,
 	tableOptions runtime.Object) (*metav1.Table, error) {
+	return t.tableConverter.ConvertToTable(ctx, object, tableOptions)
+}
+
+// printHandler registers the column definitions and actual formatter functions
+func printHandler(h printers.PrintHandler) {
+	columnDefinitions := []metav1.TableColumnDefinition{
+		{Name: "Name", Type: "string", Format: "name", Description: metav1.ObjectMeta{}.SwaggerDoc()["name"]},
+		{Name: "User", Type: "string", Description: "User is the owner of the token"},
```

Comment:
I'd consider User to be column with Priority 1, as most users are not admins.

Reply:
Fair point. Will change.

Reply:
done. commit 6434fb02ab93bbf500610c0d6214ca9daab4515e